pkg/controller/volume/expand/cache/volume_resize_map.go

```diff
@@ -68,6 +68,10 @@ type PVCWithResizeRequest struct {
 	CurrentSize resource.Quantity
 	// Expended volume size
 	ExpectedSize resource.Quantity
+	// ReQueued indicates that the PVC is being
+	// reprocessed for resize because of either some error
+	// or via populaor loop.
+	ReQueued bool
```

Comment:
`ReQueued` -> `Requeued`? Either it's wrong here or in `requeued` parameters in AddPVCUpdate above - it should be `reQueued` there, following the same logic. I'd vote for `Requeued`, IMO it's one English word.